packages/cloud-cognitive/src/components/ButtonMenu/ButtonMenu.js

```diff
@@ -116,6 +118,11 @@ ButtonMenu.propTypes = {
    */
   iconDescription: Button.propTypes.iconDescription,
 
+  /**
+   * The three types the menu button supports: primary, tertiary and ghost.
+   */
+  kind: Button.propTypes.kind,
```

Comment:
```suggestion
  kind: PropTypes.oneOf(['primary', 'tertiary', 'ghost']),
,
```

Reply:
@elycheea Many thanks for reviewing this. Proposed changes were implemented.